source/blog/2017-11-05-vdsm-setup-networks-is-now-much-quicker.html.md

```diff
@@ -0,0 +1,40 @@
+---
+title: VDSM setupNetworks is now much quicker
```

Comment:
Perhaps a more generalized title? 'Setting up multiple networks is going to be much faster in oVirt 4.2'

Reply:
Done